build/checkstyle/suppressions.xml

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE suppressions PUBLIC "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+        "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+<suppressions>
+    <suppress files="[\\/]src[\\/]test[\\/].*" id="MethodNameRegular" />
+    <suppress files="[\\/]src[\\/]main[\\/].*" id="MethodNameTest" />
+</suppressions>
```

Comment:
also, could you explain each suppress rule in comment, so we know what is suppressed for what?

Thanks